node/src/BaseClient.ts

```diff
@@ -3474,6 +3476,35 @@ export class BaseClient {
         return this.createWritePromise(createZMPop(key, modifier, count));
     }
 
+    /**
+     * Returns the distance between `member1` and `member2<` saved in the geospatial index stored at `key`.
+     *
+     * See https://valkey.io/commands/geodist/ for more details.
+     *
+     * @param key - The key of the sorted set.
+     * @param member1 - The name of the first member.
+     * @param member2 - The name of the second member.
+     * @param geoUnit - The unit of distance measurement - see {@link GeoUnit}.
```

Comment:
we don't mention the default unit if it's not specified.  
https://github.com/valkey-io/valkey-glide/blob/3f8f62d9acfc1760f03d318058885feab6f6571f/java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java#L209